ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx

```diff
@@ -67,7 +67,7 @@ export function UserCreator() {
 
     const res = await createUser({ input });
     if (res.error) {
-      toast.error('Failed to create user');
+      toast.error(`Failed to create user: ${res.error.graphQLErrors?.[0]?.message}`);
```

Comment:
Can we append something like "Please contact support" and if possible in Sonner, link to `https://app.inngest.com/support`? we can also let it have longer duration as well so someone has time to click on the link.

Reply:
Done :) https://github.com/inngest/inngest/pull/830/commits/d9375c334e4a563c8e5d90729d7448a3f174ff2e